src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -45,6 +45,9 @@ using namespace std::chrono_literals;
 
 namespace ripple {
 
+// The maximum number of transactions to have in the job queue.
+int const max_transactions = 1000;
```

Comment:
Since this constant is only used one place in the file, consider reducing its scope to just where it is needed.  Here's an example of what I have in mind: https://github.com/scottschurr/rippled/commit/99f3bfcd240b6d66ed698266d4024646c659369e

BTW, I like providing a name for the magic number.  Good work with that.

Reply:
Fixed

Reply:
The constant value changes decrease the cost by 10x for peers to send transactions to one another, while leaving the relative for all other activities the same.